include/boost/algorithm/string/formatter.hpp

```diff
@@ -83,7 +83,7 @@ namespace boost {
         template<typename RangeT>
         inline detail::empty_formatF< 
             BOOST_STRING_TYPENAME range_value<RangeT>::type>
-        empty_formatter(const RangeT&)
+        empty_formatter(const RangeT& Input)
```

Comment:
I don't think the parameter is used; so naming it here would probably generate other warnings; something like "input not used"

Reply:
Mmm, true. I didn't observe such a warning but I'm sure they would come up. The docstring "parameter not found" doesn't like unnamed arguments as far as I can tell.

Which warning is preferred? Should I revert this one?

Reply:
I'd rather have a docstring warning than a compiler warning.

Reply:
The other way to deal with it is to add `(void) Input;` to the body of the function.

Reply:
Ok, reverted in c34713b!

Reply:
thanks!